plugins/flashrom/fu-flashrom-internal-device.c

```diff
@@ -52,6 +52,7 @@ fu_flashrom_internal_device_prepare (FuDevice *device,
 		FuFlashromDevice *parent = FU_FLASHROM_DEVICE (device);
 		struct flashrom_flashctx *flashctx = fu_flashrom_device_get_flashctx (parent);
 		gsize flash_size = fu_flashrom_device_get_flash_size (parent);
+		struct flashrom_layout *layout;
```

Comment:
did you forget to push more?

Reply:
No, BtG + IFWI used all the brain I had available today, will do it tomorrow!

Reply:
Maybe it makes sense to set the pointer to NULL initially?